infrastructure/applications/gateway/main.tf

```diff
@@ -19,15 +14,15 @@ module "lambda" {
   docker_repository_name = "gateway"
   docker_tag             = terraform.workspace
   role_arn               = data.aws_iam_role.lambda.arn
+  memory_size            = 1024
```

Comment:
kind of hoping that giving it more memory would improve cold start, but it seems that cold start with docker images aren't great.

need more investigation, 1gb isn't a lot so it's ok to keep it I think. we could raise all services from 512mb to 1gb to be honest